### ISSUE TITLE
Quest #4262 bugged 

### ISSUE THREAD

**Issue by shobits1** [dalaranwow/dalaran-wow#5287]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Where -> Searing Gorge on Algalon
When -> *** not sure maybe always
What -> Quest: "Overmaster Pyron" https://wowgaming.altervista.org/aowow/?quest=4262
Why -> I can't slay Overmaster Pyron,, I think he is under the mountain

**Expected behaviour**: Tell us what should happen instead
I should find and slay Overmaster Pyron as specified in https://wowgaming.altervista.org/aowow/?quest=4262

**Steps to reproduce the problem**:
not sure if it needs step,, as I come to the place where I should find Pyron but couldn't.. at first I thought that he was killed by other player so I waited him to respawn but nothing happened.... then I tried to target him using "/tar Overmaster Pyron" and there he is but still couldn't find him,, put a star on him but no he is not showing.... since I'm hunter I send my pet to attack him but the pet keeps cercling the mountain then returnning to me.... this is why I think he is under (as you can see in the pic)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wowgaming.altervista.org/aowow/?quest=4262

<a href="https://ibb.co/j73cvc"><img src="https://preview.ibb.co/erg81H/Wo_WScrn_Shot_020518_194052.jpg" alt="Wo_WScrn_Shot_020518_194052" border="0"></a><br /><a target='_blank' href='https://imgbb.com/'>bbcode link image</a><br />

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
I've checked it, and the NPC spawns where it should, which is not the spot where you are in that picture.

Cheers, closing up.
